docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md

```diff
@@ -24,7 +24,7 @@ Additionally, the following package manager are required to build LORIS:
 
 * NodeJS 16.10.0 (or higher)
 
-* NPM
+* NPM 5.7.0 (or higher)
```

Comment:
Hi @laemtl, I think now could be a good time to update the recommended NodeJS version to the current LTS (long-term-support) version and that uses the npm version 8.19.2.